Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN set -eux; \
     	curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/${kubectlArch}/kubectl && \
     	chmod +x ./kubectl && \
     	mv ./kubectl /usr/local/bin/kubectl && \
-    	kubectl version --client=true
+        which kubectl
 
 EXPOSE 8001
```

Comment:
We expose port 8001 here... but in the kubernetes-credential-manager.sh the "kubectl proxy" command is removed ... that exposes port 8001. Do we need this still?

Reply:
Agreed. Not needed. Removing.